### DISCUSSION TITLE
Support for issue related task is scheduled 2 time automaticaly.

### DISCUSSION THREAD

**Discussion by Autoeps** [conductor-oss/conductor#137] [General]
I have Created a workflow, where there are some task, like  "Inline Task" and "Set Variable" task, Set_Name_ref1 has 2 attempts. 
![image](https://github.com/conductor-oss/conductor/assets/167202483/e163f7bb-e149-4571-ac3c-7f1281ffab94)
Retried Task has 2 attempt in workflow it has only one task. 
**Attempt 1**
![image](https://github.com/conductor-oss/conductor/assets/167202483/4803d24c-8461-4f4e-bde0-dc665e0308d9)

**Attempt 2**
![image](https://github.com/conductor-oss/conductor/assets/167202483/679485d8-40d9-44a7-bef4-296b307bc46f)

**json**
{
  "taskType": "SET_VARIABLE",
  "status": "COMPLETED",
  "inputData": {
    "emailStatus": "Retry_RetryLimitCheck",
    "currentDate": "2024-04-16 07:24:28",
    "continuesLoop": true
  },
  "referenceTaskName": "Set_Name_ref1",
  "retryCount": 0,
  "seq": 5,
  "correlationId": "661e2766037f4e2e08fc292c",
  "pollCount": 0,
  "taskDefName": "WF variable for continuesLoop, currentDate, emailStatus",
  "scheduledTime": 1713252268628,
  "startTime": 1713252268628,
  "endTime": 1713252268669,
  "updateTime": 1713252268634,
  "startDelayInSeconds": 0,
  "retried": false,
  "executed": true,
  "callbackFromWorker": true,
  "responseTimeoutSeconds": 0,
  "workflowInstanceId": "f0ce0fa8-c361-4ba4-8e04-3404df3c3c8a",
  "workflowType": "CDO_CRM_SendLimit",
  "taskId": "db93ef79-afce-4005-981f-88e72fc164a6",
  "callbackAfterSeconds": 0,
  "outputData": {},
  "workflowTask": {
    "name": "WF variable for continuesLoop, currentDate, emailStatus",
    "taskReferenceName": "Set_Name_ref1",
    "inputParameters": {
      "continuesLoop": true,
      "currentDate": "${GetCurrentDate.output.result.date}",
      "emailStatus": "${workflow.input.emailStatus}"
    },
    "type": "SET_VARIABLE",
    "startDelay": 0,
    "optional": false,
    "asyncComplete": false,
    "permissive": false
  },
  "rateLimitPerFrequency": 0,
  "rateLimitFrequencyInSeconds": 0,
  "workflowPriority": 0,
  "iteration": 0,
  "subworkflowChanged": false,
  "queueWaitTime": 0,
  "loopOverTask": false,
  "taskDefinition": null
}

**Issue here is that why SET_VARIABLE task has Scheduled 2 time and has Retried Task 2 Attempt and both the attempt is sucesses and both the attempt sequence is 0 .** 

![image](https://github.com/conductor-oss/conductor/assets/167202483/53fc9dad-2dad-4a56-9b57-fb236cd6aeb9)

**does anyone have Idea realted to such issue.**

**Comment by JCHacking** [comment]
How many instances of Conductor did you have active at that time?
When there is more than one, these problems can occur, so it is necessary to use blocking systems (can be configured for Conductor).
